src/lib/mongoBackend/mongoRegistrationGet.cpp

```diff
@@ -79,12 +79,14 @@ static void setDescription(ngsiv2::Registration* regP, const mongo::BSONObj& r)
 *
 * setProvider -
 */
-static void setProvider(ngsiv2::Registration* regP, const mongo::BSONObj& r)
+static void setProvider(ngsiv2::Registration* regP, const mongo::BSONObj& r, const char* dbLabel)
```

Comment:
Function is overparametriced, `dbLabel` is not needed at the field used for provier is not a parameter but fixed by the database model.

Reply:
Fixed in 5d851ef